src/components/WaveForm.tsx

```diff
@@ -0,0 +1,81 @@
+"use client";
+
+import dynamic from "next/dynamic";
+import {
+  CSSProperties,
+  PointerEvent,
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+} from "react";
+
+const WavesurferPlayer = dynamic(() => import("@wavesurfer/react"), {
+  ssr: false,
+  loading: () => <div style={{ height: getHeight() }} />,
+});
+
+type Props = {
+  percentage: number;
+  src: string;
+};
+
+export function WaveForm({ percentage, src }: Props) {
+  const containerRef = useRef<HTMLDivElement>(null);
+  const wavesurferRef = useRef<any>(null);
```

Comment:
여기 wavesurferRef의 useRef가 any로 설정된걸 해결하고 싶어서 해결 해봤습니다!
이 파일 상단에 `import WaveSurfer from "wavesurfer.js";` 추가해주시고

```suggestion
  const wavesurferRef = useRef<WaveSurfer | null>(null);
```
요렇게 바꿔주심 될 것 같슴다!

Reply:
사실 이 부분은 외부 코드를 그대로 참조한거라 사용될 지 아닐지도 확실하지 않아 일단 기능만 구현된 채로 냅두었던건데 any 타입이 있었군요..😓😓 더욱 깔끔하고 좋은 것 같습니다 !